File: guessNum.c


#include<stdio.h>
#include<stdlib.h>
#include <time.h> 


int main(){
    srand(time(NULL));
    int rNum = (rand()%20)+1;
    int iNum;
    int counter = 0;


    do{
        printf("Enter a Number between 0 and 20:\n");
        scanf("%i", &iNum);
        if (iNum > rNum){
            counter++;
            printf("Too High\n");
            printf("You have %i guesses.\n", 5 - counter);
        }
        if (iNum < rNum){
            counter++;
            printf("Too Low\n");
            printf("You have %i guesses.\n", 5 - counter);
        }
        if (iNum == rNum){
            printf("You win\n");
            break;
        }
    }while(counter < 5);
    if(counter >= 5){
        printf("You Lose.\n");
    }
}

File: guessNum.py

import random

rNum = random.randint(1,20)
counter = 0
while counter < 5:
    print("You have", 5-counter,"guesses")
    gNum = int(input("Enter a number between 1 and 20: "))
    if gNum < rNum:
        counter+=1
        print("Too Low")
    if gNum > rNum:
        counter+=1
        print("Too High")
    if gNum == rNum:
        print("You Win")
        break
if counter >= 5:
    print("You Lose")

    






    